d2/src/main/java/com/linkedin/d2/discovery/stores/zk/ZooKeeperEphemeralStore.java

```diff
@@ -476,6 +476,12 @@ private void setRawClientTrackingNode()
     String rawD2ClientTrackingPath = D2Utils.getRawClientTrackingPath();
     try
     {
+      if (rawD2ClientTrackingPath.contains("post-commit"))
```

Comment:
adding delimiters around it "**-**post-commit**-**" feels better just in case some app names accidentally match it, for example "blogpost-committee"